packages/forms/backend/sirius-components-collaborative-forms/src/main/java/org/eclipse/sirius/components/collaborative/forms/handlers/EditTextfieldEventHandler.java

```diff
@@ -101,7 +101,11 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
             }
             if (status instanceof Success success) {
                 payload = new SuccessPayload(formInput.id(), success.getMessages());
-                changeDescription = new ChangeDescription(ChangeKind.SEMANTIC_CHANGE, formInput.representationId(), formInput);
+                if (success.getChangeKind() == null || success.getChangeKind().isBlank()) {
```

Comment:
Currently, a change from a Textfield in the view details panel returns a Success with a blank `changeKind` field.
Maybe because we knew that this change could only be semantic, the returned ChangeDescription has a changeKind forced to `ChangeKind.SEMANTIC_CHANGE`.
In our case, we introduce two new changes that are no longer semantic since they occur on the representation metadata.
Hence, we have to leverage the `changeKind` of the success status and return this change kind in the ChangeDescription.
I admit that the nullable check might be removed.

Reply:
nullable check removed